migrations/20200304235514-add-item-associations.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    /*
+      Add altering commands here.
+      Return a promise to correctly handle asynchronicity.
+
+      Example:
+      return queryInterface.createTable('users', { id: Sequelize.INTEGER });
+    */
+    return queryInterface.sequelize.transaction(function(transaction) {
+      return queryInterface.removeColumn("Items", "categoryId", {transaction})
+        .then(() => queryInterface.removeColumn("Items", "restaurantId", {transaction}))
+        .then(() => queryInterface.addColumn("Items", "CategoryId", {
+            type: Sequelize.INTEGER,
+            references: {
+              model: "Categories",
+              key: "id"
+            }
+          }, {transaction}))
+        .then(() => queryInterface.addColumn("Items", "RestaurantId", {
+            type: Sequelize.INTEGER,
+            references: {
+              model: "Restaurants",
+              key: "id"
```

Comment:
And here we tell the database that the RestaurantId column in items references the id column of the Restaurants table.